lua/entities/gmod_tardis_interior/modules/sh_security.lua

```diff
@@ -29,8 +29,11 @@ end)
 
 ENT:AddHook("CanUsePart","security",function(self,part,ply)
 	if self:GetData("security", false) and (ply~=self:GetCreator()) then
-		if part.ID == "door" then return end
+		-- Default to false if not exists. This is to not break any extensions' parts
+		part.BypassIsomorphic = part.BypassIsomorphic or false
+
+		if part.ID == "door" or part.BypassIsomorphic then return end
```

Comment:
The door part should have BypassIsomorphic set to true instead of this explicit bypass here
(this is a copy of the comment by Dr.Matt at https://github.com/MattJeanes/TARDIS/pull/383/files#r588196768)